MHRA/products
# Clear cookies by setting path and domain

# Clear all cookies

Relates to bug #498

![](https://media.giphy.com/media/HGe4zsOVo7Jvy/giphy.gif)

### Acceptance Criteria

- [ ] All cookies are cleared

### Testing information

1) Accept cookies. 
2) Go to cookies policy page and disallow cookies
3) Refresh page
4) Check cookies in dev tools - they should be empty

### Pre-flight Checklist


### Pre-merge Checklist


[testing strategy]: https://github.com/MHRA/products/blob/master/docs/principles/testing.md "MHRA/products testing strategy"
